packages/machine-extractor/package.json

```diff
@@ -1,14 +1,13 @@
 {
-  "name": "xstate-parser-demo",
+  "name": "@xstate/machine-extractor",
   "version": "0.5.8",
-  "main": "dist/xstate-parser-demo.cjs.js",
+  "main": "dist/xstate-machine-extractor.cjs.js",
   "author": "Matt Pocock",
   "license": "MIT",
   "scripts": {
     "test": "jest src",
     "lint": "tsc",
-    "prepare": "patch-package && yarn build",
-    "build:prod": "(cd ../ && preconstruct build)"
+    "prepare": "patch-package && yarn build"
```

Comment:
this patch is most likely not relevant for the nested package - usually we only want such things at the root, so perhaps this is outdated and not needed at all?

Reply:
`build` script is also not defined here so probably this `prepare` script can be dropped